public/README_notes.md

```diff
@@ -14,6 +14,7 @@ widget components are defined modularly in their own component files.
 
 the entire config is loaded from a json file requested from the server.
 
+
```

Comment:
Nit: This extra line was probably not intentional?

Reply:
actually I didnt need most of those notes anymore, removed.